### PR TITLE
added tagline

### PR DESCRIPTION
**Pull request by Fayyr**

### CODE REVIEW

File: bl-content/databases/categories.php [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?php defined('BLUDIT') or die('Bludit CMS.'); ?>
+{"computers":{"name":"Computers","template":"","description":"elit a mi vehicula, ac rhoncus nunc malesuada. Aliquam ac rutrum nisl, ut elementum felis. Nulla nisl mauris, lobortis ut tristique in, porttitor non ipsum. Nulla dignissim ac mauris a iaculis. Curabitur pharetra ac enim a sodales. Sed iaculis leo tincidunt efficitur eleifend. Nulla finibus pharetra laoreet. Duis eget mauris ut enim blandit sodales. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.","list":[]},"general":{"name":"General","description":"","template":"","list":[]},"music":{"name":"Music","description":"","template":"","list":[]},"videos":{"name":"Videos","description":"","template":"","list":[]}}
```
Comment (someshkoli) [inline bl-content/databases/categories.php]: Remove the database contents. Its not required in the main codebase. Its auto generated

File: bl-content/databases/pages.php [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
...etup ","description":"","username":"admin","tags":[],"type":"published","date":"2020-09-24 20:12:43","dateModified":"","position":6,"coverImage":"Infante50-bs.jpg","category":"","md5file":"c7ff24ba5971e59ffea8b679d50b3e7a","uuid":"4b33ac6f56ca4c10e462fa46667ca64e","allowComments":true,"template":"","noindex":false,"nofollow":false,"noarchive":false,"custom":[false]},"lorem-ipsum":{"title":"Cool mask bruv","description":"","username":"admin","tags":[],"type":"published","date":"2020-09-24 14:50:57","dateModified":"2020-09-24 20:15:44","position":5,"coverImage":"pexels-sebastiaan-stam-1097456.jpg","category":"","md5file":"1c0ae4b9b3b62a62112dc2b103f2f9ca","uuid":"126ebbc737a9e390f6fceef5db4f93ae","allowComments":true,"template":"","noindex":false,"nofollow":false,"noarchive":false,"custom":[false]},"conus":{"title":"Contact Us","description":"","username":"admin","tags":[],"type":"static","date":"2020-09-23 17:55:00","dateModified":"2020-09-23 18:39:16","position":2,"coverImage":"","category":"","md5file":"60e743f6af96ea9494e0fc21570b4351","uuid":"2e7e607135d6a533b2478db9a52729d5","allowComments":true,"template":"","noindex":false,"nofollow":false,"noarchive":false,"custom":[false]}}
```
Comment (someshkoli) [inline bl-content/databases/pages.php]: same

File: bl-content/databases/plugins/about/db.php [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?php defined('BLUDIT') or die('Bludit CMS.'); ?>
+{
+    "position": 1,
+    "label": "About",
+    "text": "We at Decrypt aim to write the best stuff"
+}
```
Comment (someshkoli) [inline bl-content/databases/plugins/about/db.php]: Why is the auto generated database required for these changes ? :) These are static contents right ?

**Review by someshkoli — CHANGES_REQUESTED** [review]
remove bl-content folder from the pr -_+

**Comment by Fayyr** [comment]
> 
> 
> remove bl-content folder from the pr -_+

about page ka content usmei hi hai , i didnt write alagse code for about page , i used a static page and usme html daal diya :c

**Comment by someshkoli** [comment]
> > remove bl-content folder from the pr -_+
> 
> about page ka content usmei hi hai , i didnt write alagse code for about page , i used a static page and usme html daal diya :c

it should be there somewhere in static contents

**Comment by Fayyr** [comment]
> 
> 
> > > remove bl-content folder from the pr -_+
> > 
> > 
> > about page ka content usmei hi hai , i didnt write alagse code for about page , i used a static page and usme html daal diya :c
> 
> it should be there somewhere in static contents

bl-content\pages\about\index.txt hai

**Comment by someshkoli** [comment]
Add about view in kernel/admin/views/about.php

**Comment by Fayyr** [comment]
> 
> 
> Add about view in kernel/admin/views/about.php

woh admin panel ka about view hai na ? websiite ka alag hai

**Comment by Fayyr** [comment]
> 
> 
> > Add about view in kernel/admin/views/about.php
> 
> woh admin panel ka about view hai na ? websiite ka alag hai

host karne ke baad bhi add karsakte na about page ? ill just pass the html through the text edit editor

**Comment by someshkoli** [comment]
> > Add about view in kernel/admin/views/about.php
> 
> woh admin panel ka about view hai na ? websiite ka alag hai

umm maybe
theek hai filhal ye sab contents nikal yaha se bl-content wala folder and pr khali kar thoda

**Review by someshkoli — APPROVED** [review]

**Pull request merged**